test/integration/fatality_notice_test.rb

```diff
@@ -15,7 +15,6 @@ class FatalityNoticeTest < ActionDispatch::IntegrationTest
       visible: false
     )
 
-    assert_component_parameter("title", "context", "Operations in Zululand")
```

Comment:
Should this deleted line be replaced with an `assert_has_component_title` test, as added in `choose_sign_in_test.rb`?

Reply:
Updated with fixed test.